### PR TITLE
Fix checkdb and external ssl configuration

### PR DESCRIPTION
**Pull request by anddann**
This PR fixes the following issues:
- `docker-entrypoint.sh` checks for `checkdb.py` return value 2, however `checkdb.py` never returns 2. Thus, I check in `checkdb.py` if an initial db exists
- `docker-entrypoint.sh` actually copies the nginx ssl file into the container. However, when using an external ssl-reverse-proxy, `nginx` *inside* the container must not be adapted. IMHO: that is the point of using an external ssl-reverse proxy 
PS: If you agree with this PR; we should create a valid and working *tag* to indicate the working version and maybe clean up the old *non-working* tags?

### CODE REVIEW

File: docker-entrypoint.sh [hunk 1]
```diff
@@ -48,8 +48,8 @@ elif grep -q "wss://" "/taiga/conf.json"; then
 fi
 
 # Reinitialize nginx links
-rm /etc/nginx/sites-enabled/*
-if [ "$TAIGA_SSL_BY_REVERSE_PROXY" = "True" ] || [ "$TAIGA_SSL" = "True" ]; then
+if [ "$TAIGA_SSL" = "True" ]; then
+  rm /etc/nginx/sites-enabled/*
```
Comment (ajira86) [inline docker-entrypoint.sh]: Why this change ?

Comment (anddann) [reply docker-entrypoint.sh]: You are correct, the rm statement actually belongs in the line above.
Concerning  removing  `$TAIGA_SSL_BY_REVERSE_PROXY" = "True"` from the if condition, lets assume the following setup, which I use:
- I have a server
- the server runs nginx and redirects all incoming traffic to `127.0.0.1:8080`
- the server's nginx reverse proxy, does all SSL handling. It is the external reverse proxy that is responsible for the certificates, keys, handshake, you name it,...
- on port `127.0.0.1:8080` the taiga container listens to requests and forwards to it's internal port 80.
Thus, all TLS stuff is done by the server's nginx installation, the taiga container knows nothing about SSL, excepts the hostname.
IMHO: that's the point of an external reverse proxy, otherwise one have to set up a all SSL stuff within the container.
Please correct me if I'm wrong.

Comment (ajira86) [reply docker-entrypoint.sh]: Ok, thanks for your explanation, this sound correct.
